aws-android-sdk-cognitoidentityprovider/src/main/java/com/amazonaws/mobileconnectors/cognitoidentityprovider/CognitoUserSession.java

```diff
@@ -103,16 +118,36 @@ public boolean isValid() {
     }
 
     /**
-     * Returns true if this session for the threshold set in {@link CognitoIdentityProviderClientConfig#refreshThreshold}.
+     * Returns true if this session for the threshold set in
```

Comment:
"if this session is valid for the threshold..."

Reply:
Fixed